sources/packages/web/src/views/institution/Reports.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <full-page-container>
+    <template #header
+      ><header-navigator title="Institution" subTitle="Reports" />
+    </template>
+    <Reports title="Institution" :reportList="reportList" />
```

Comment:
Please use it as `reports` and the property as `report-list`, as below.
We have mixed patterns but the below-mentioned one is preferred.
```vue
<reports title="Institution" :report-list="reportList" />
```